pkg/interfaces/mocks/StoreClient.go

```diff
@@ -65,31 +64,51 @@ func (_m *StoreClient) RetrieveFromStore(appServiceKey string) ([]contracts.Stor
 }
 
 // Store provides a mock function with given fields: o
-func (_m *StoreClient) Store(o contracts.StoredObject) (string, error) {
+func (_m *StoreClient) Store(o interfaces.StoredObject) (string, error) {
 	ret := _m.Called(o)
 
 	var r0 string
-	var r1 error
+	if rf, ok := ret.Get(0).(func(interfaces.StoredObject) string); ok {
+		r0 = rf(o)
+	} else {
+		r0 = ret.Get(0).(string)
+	}
 
-	if rf, ok := ret.Get(0).(func(contracts.StoredObject) (string, error)); ok {
-		r0, r1 = rf(o)
+	var r1 error
+	if rf, ok := ret.Get(1).(func(interfaces.StoredObject) error); ok {
+		r1 = rf(o)
 	} else {
-		r1 = ret.Error(0)
+		r1 = ret.Error(1)
 	}
```

Comment:
These changes are causing the panic in the unit test. The old code should be restored. 

This is another place that the generated code is not correct and hand fixed. We should add a comment to this effect in this block.

Change from alias `contracts` to `interfaces` is good.